gitpod-io/gitpod
# Update k8s dependencies to v0.24.3

## Description


- Update k8s dependencies to v0.24.3
- Remove empty `status` and `resource` definitions from manifests

## Related Issue(s)

fixes https://github.com/gitpod-io/gitpod/issues/8529
fixes https://github.com/gitpod-io/gitpod/issues/11136

## How to test
- Check workspaces work as usual
- Check the generated manifests do not contain empty `status` or `resource` definitions

```
docker run --entrypoint sh --rm \
  eu.gcr.io/gitpod-core-dev/build/installer:aledbf-k8s24.11 -c "cat /app/installer" >gitpod-installer

chmod +x gitpod-installer

cat <<EOF >values.yaml
apiVersion: v1
domain: fake.gitpod.io
EOF

./gitpod-installer render --use-experimental-config --config values.yaml > manifests.yaml
```

## Release Notes
```release-note
NONE
```

## Werft options:
